Sakura/Storage/cache.py

```
@@ -46,4 +46,42 @@ async def delete_cache(key: str):
+    Returns:
+        The cached message text, or None if not found or on error.
+    """
+    if not state.valkey_client:
+        return None
+
+    try:
+        last_message = await state.valkey_client.get(f"last_message:{user_id}")
+        if last_message:
+            logger.info(f"Retrieved last message for user {user_id}")
+            return last_message
+        return None
```

Ruff (0.13.2)

Consider moving this statement to an `else` block

(TRY300)